src/libs/render-utils.tsx

```diff
@@ -5,7 +5,7 @@ import CopyTextButton from 'components/common/CopyTextButton';
 import InfoHoverButton from 'components/common/InfoHoverButton';
 import { DatasetSummaryModel, SnapshotSummaryModel } from '../generated/tdr';
 
-const cloudPlatforms = { gcp: 'Google Cloud Platform', azure: 'Microsoft Azure' };
+export const cloudPlatforms = { gcp: 'Google Cloud Platform', azure: 'Microsoft Azure' };
```

Comment:
For a potential improvement for the future, I wonder if you could put the regions in this constant? Then you could do some filtering and mapping to get regions for a particular cloudPlatform, rather than needing to do conditionals. I think that might be more powerful if you end up needing to add another cloud platform to simplify the number of places you would need to add. (I think you might be able to get it to the point where the only place you would need to modify is here, and every bit of the UI would automatically update to use that new cloudPlatform)

Reply:
The future is now! updated this by moving the constant out of the utils and into the CONSTANTS file where it belongs!